docs/AppendixA-FAQ.md

```diff
@@ -1 +1,35 @@
 # Appendix A - FAQ
+
+This section covers some of the most commonly encountered errors when running 
+the insertion scripts. They are divided into 3 sections depending on whether 
+the errors are related to the LORIS-MRI installation, LORIS front-end modules 
+setup, or LORIS-MRI scripts.
+
+The user is referred to the [README.md](../README.md) which includes a 
+post-installation verification section.
```

Comment:
this link should eventually point to the docs/section 2.4 where the post installation details/troubleshooting will be housed.